libs/execution_context/ebpf_core.c

```diff
@@ -31,6 +31,24 @@ _ebpf_core_map_delete_element(ebpf_map_t* map, const uint8_t* key);
 
 #define EBPF_CORE_GLOBAL_HELPER_EXTENSION_VERSION 0
 
+static ebpf_helper_function_prototype_t _ebpf_map_helper_function_prototype[] = {
+    {1,
+     "ebpf_map_lookup_element",
+     EBPF_RETURN_TYPE_PTR_TO_MAP_VALUE_OR_NULL,
+     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}},
+    {2,
+     "ebpf_map_update_element",
+     EBPF_RETURN_TYPE_INTEGER,
+     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_VALUE}},
+    {3,
+     "ebpf_map_delete_element",
+     EBPF_RETURN_TYPE_INTEGER,
+     {EBPF_ARGUMENT_TYPE_PTR_TO_MAP, EBPF_ARGUMENT_TYPE_PTR_TO_MAP_KEY}}};
+
+static ebpf_program_info_t _ebpf_global_helper_program_info = {{"global_helper", NULL, {0}},
```

Comment:
From reading some ebpf blogs and docs, I think the term "general" is more appropriate than "global".
(It's also more consistent with using "specific" for things that are program type specific, since general and specific are antonyms whereas global and local are antonyms)

Reply:
Agree. Will change global to general or generic, as much as possible.

Reply:
replaced global_helper with generic_helper.

Reply:
See https://ell.stackexchange.com/questions/16224/what-is-the-difference-between-general-and-generic

I think "general" is more correct than "generic"

Reply:
Another reason to avoid "generic" is that it also means something else in programming languages, such as is referred to in https://www.geeksforgeeks.org/generics-in-c/ and https://en.wikipedia.org/wiki/Generic_programming etc.

Reply:
changed to 'general'.